app/src/main/java/com/kickstarter/ui/activities/compose/ChangePasswordScreen.kt

```diff
@@ -147,9 +145,9 @@ fun ChangePasswordScreen(
         },
         snackbarHost = {
             SnackbarHost(
-                hostState = scaffoldState.snackbarHostState,
+                hostState = snackbarHostState,
```

Comment:
Example for snackbar states, more like this will come on follow up tickets.
`scafoldState` is gone, now we should hold `SnackbarHostState()`, and pass it to the snackbarhost

To show the message instead of `message`, `visuals.message`